charts/admission-controller/values.yaml

```diff
@@ -195,6 +195,15 @@ webhook:
       pullPolicy:
   # The service name for Webhook deployment
   name: webhook
+
+  # Vulnerability Management extension settings
+  vm:
+    enabled: true
+    # NOTE: If you're installing ClusterScanner via ClusterShield
+    # the clusterScannerEndpoint will be in the form `${cluster-shield-installation-fullname}-cluster-scanner:9999`
+    # Port is always 9999
+    clusterScannerEndpoint: "CHANGEME:9999"
```

Comment:
Is there a default `cluster-shield-installation-fullname` that we can use instead of CHANGEME? 

I am assuming (CC @guidobonomi ) that installing ClusterScanner via ClusterShield is the only option, because we are not releasing the latest changes for ClusterScanner via standalone chart? If we, I would remove the conditional "If you are installing...".

Reply:
@airadier no clustershield is not the only option, and no there is unfortunately no sensible default since the service name depends on the (helm) installation name